plans/inhibit-if-kmods-is-not-supported/centos8/main.fmf

```diff
@@ -0,0 +1,14 @@
+discover:
+    how: fmf
+    # TODO filter inheritance doesn't work
+    filter:
+        - 'tag: centos8'
+        - 'tag: inhibit-if-kmods-is-not-supported'
+provision:
+    how: container
+    image: convert2rhel/centos8
+    pull: false
+prepare:
+    how: ansible
+    playbook:
+        - tests/integration/inhibit-if-kmods-is-not-supported/ansible/setup.yml
```

Comment:
Hm... Seems that the `unshare` command is available in `podman` but not in `docker`? @thrix any idea?

Reply:
I would expect running something like `docker exec convert2rhel/centos8 ansible-playbook ....`

Reply:
prepare has different paths @ZhukovGreen as I showed in the jira ticket ...

Reply:
the feature to have the paths as one would expect .. there is an issue for that

Reply:
yeah, the docs we updated is misleading again, due to this problem

Reply:
docker is not supporter, only podman :)

Reply:
we never tested with docker ... let's stick to podman

Reply:
Isn't the problem as I described OAMG-4515  ?

Reply:
> I would expect running something like docker exec convert2rhel/centos8 ansible-playbook ....

We do not want to run ansible inside docker. That would require installing Ansible in the container, would not work for older RHELs, etc. We run ansible on the "localhost" using podman Ansible connection plugin to connect to the container.

Reply:
@thrix thx for sharing this. You were right, this is due to docker issue. After installation of podman, it seems almost working. As I understood tmt is now only capable to run particular playbooks, rather than roles?

Reply:
I would vote for running the ansible playbook inside the container. It seems wrong to me to rely on the host. For example I have Debian and docker. Now I am in trouble to continue. Is it the same behavior for VMs?

Reply:
> I would vote for running the ansible playbook inside the container.

There are use cases where the control host must be outside (like restart of the system). So there is a reason why we do it like this. Of course, that does not apply to the case of containers. Still I am not sure we could easily run ansible for RHEL6 for example, but from outside it should work ...

If you think running ansible from inside is better, we can have it, but it should be optional and we can discuss what the default should be if that is done

> It seems wrong to me to rely on the host.
Well you already rely on host with tmt right? :)

> I have Debian and docker. 

But you know, there is also an ansible docker plugin :) so it just needs few patches to make the "backend" 
use docker instead ... `https://docs.ansible.com/ansible/latest/collections/community/docker/docker_connection.html#ansible-collections-community-docker-docker-connection`

> Now I am in trouble to continue.

Try to replicate the podman provision plugin to docker one, it should not be hard.

>  Is it the same behavior for VMs?

Yes, it is.

Reply:
@thrix Thx Miroslav, I understand your point, I'll try to workaround it somehow

Reply:
thanks for the discussion, it is good feedback to have!

Reply:
OK, I was able to achieve the possibility to run playbooks inside the container. But without TMT yet. It seems podman plugin doesn't work as expected and runs the playbook against the host rather than against the container. So preparation in this sense relates to the host and not to the container. Submitted a ticket https://github.com/psss/tmt/issues/621